ntbs-service/DataAccess/ItemRepository.cs

```diff
@@ -26,23 +26,23 @@ public async Task AddAsync(T item)
         {

```

Comment:
I noticed that we weren't adding the correct audit type here - I've done some minor other tidying of auditing as part of this commit